ui/shared/InputFilter.jsx

```diff
@@ -36,10 +36,9 @@ export default class InputFilter extends React.Component {
 
   userActionListener = async event => {
     const { updateFilterText } = this.props;
-    const { input } = this.state;
 
     if (event.key === 'Enter') {
-      updateFilterText(input);
+      updateFilterText(event.target.value);
```

Comment:
This change was done because during unit testing (with the older implementation), the React Input receives entered text, but does not propagate the entered values back to `AlertView` component.

Reply:
Not sure about that.
I've reverted the changes done in this file, except the `onKeyPress={updateOnEnter ? this.userActionListener : undefined}` line. Looks like all tests are passing.

Reply:
Done.